api/build.sbt

```diff
@@ -60,22 +60,28 @@ libraryDependencies ++= Seq(
   "com.amazonaws" % "aws-java-sdk-dynamodb" % amazonawsVersion,
   "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingVersion,
   "org.apache.commons" % "commons-lang3" % apacheCommonsVersion,
-  "com.lightbend.akka" %% "akka-stream-alpakka-sqs" % "5.0.0",
-  "com.typesafe.akka" %% "akka-http" % "10.1.15",
-  "com.typesafe.akka" %% "akka-http-core" % "10.1.15",
+  "com.lightbend.akka" %% "akka-stream-alpakka-sqs" % "3.0.4",
   "com.gu" % "kinesis-logback-appender" % "1.4.2"
 )
 
-sourceDirectory in webappPrepare := (sourceDirectory in Compile).value / "resources/webapp"
+// Pin Akka versions to last available non-commercially licensed versions
+val akkaOverrides = Seq(
+  "com.typesafe.akka" %% "akka-stream" % "2.6.21",
+  "com.typesafe.akka" %% "akka-http" % "10.2.10"
+)
```

Comment:
not requesting any change or anything, just curious: why are these two in this val but not the akka-stream one ?